src/snowflake/snowpark/window.py

```diff
@@ -225,7 +249,19 @@ def order_by(
                 elif isinstance(e._expression, Expression):
                     order_spec.append(SortOrder(e._expression, Ascending()))
 
-        return WindowSpec(self.partition_spec, order_spec, self.frame)
+        ast = proto.SpWindowSpecExpr()
+        window_ast = with_src_position(ast.sp_window_spec_order_by)
+        window_ast.wnd.CopyFrom(self._ast)
+        for e in exprs:
```

Comment:
there's some code duplication here, could use helper function. For now kept as is to not introduce complexity.